Sources/Paywalls/Components/PaywallTimelineComponent.swift

```diff
@@ -207,4 +212,28 @@ public extension PaywallComponent {
 
     }
 
+    final class PartialTimelineItem: PaywallPartialComponent {
+
+        public let visible: Bool?
+        public let connector: TimelineComponent.Connector?
```

Comment:
Note: the overrides for title, description and icon are already handled internally in those components.